docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -138,40 +138,49 @@ The ability to download recordings for offline viewing will be available in a fu
 
-- Major version upgrades for customer Auth and Proxy Servers occur after the
-  first minor release (for example (=teleport.major_version=).1.0) of Teleport.
+- Major version upgrades for the Teleport Auth Service and Teleport Proxy Service on 
+  Teleport Cloud occur after the first minor release (for example (=teleport.major_version=).1.0) 
+  of Teleport.
   This typically happens within one month of a new major release of Teleport
   becoming available.
 - After the first minor release, subsequent minor and patch upgrades typically
   occur within a week.
 
-<Admonition type="note" title="Minimum agent version">
-    For major version upgrades (e.g. 12.x to 13.x), your Teleport tenant will NOT be automatically upgraded if any
-    Teleport agents connected to the cluster are not compatible (> 1 major version behind).  This is done to ensure
-    backwards compatibility. To get version information on your agents versions
-    see [here](#how-can-i-find-version-information-on-connected-agents).
+For major version upgrades (for example, 12.x to 13.x), your Teleport account is NOT 
+automatically upgraded if any Teleport agents connected to the cluster are not compatible 
+(more than one major version behind). This version check is done to ensure backwards compatibility. 
+    
+To get version information for your agents, see [How can I find version information on 
+connected agents?](#how-can-i-find-version-information-on-connected-agents).
 
-    Please reach out to support if you wish to proceed regardless of this compatibility check.
-</Admonition>
+If you want to proceed with an upgrade regardless of this compatibility check, contact 
+[Teleport support](https://goteleport.com/support/).
 
-### Are upgrades times configurable for my Teleport Enterprise Cloud tenant?
+### Are upgrades times configurable for Teleport Enterprise Cloud?
 
 Yes, you can set the window start time that works best for your organization
 for upgrades. Teleport provides a scheduled maintenance time window for any
 upgrades, patches and other required updates for all tenants. An upgrade for
 your tenant will not start earlier than the window start time you set.
 
-To set the window start time select your username in the Web UI, then the Help
-& Support option from the drop-down. The Scheduled Upgrades section within that
-page allows you to configure the window start time.
+To set the window start time, select your username in the Teleport Web UI, then 
+click **Help & Support**. Under Scheduled Upgrades, you can see the current
+maintenance window start time. Click **Edit** to set a different start
+time for scheduled maintenance.
 
 Teleport recommends subscribing to the Teleport Enterprise Cloud updates at
 [status.teleport.sh](https://status.teleport.sh) to keep up to date on
 scheduled maintenance.
 
+### When are agents automatically updated?
+
+Typically, Teleport agents are automatically updated from one to a few days after 
```

Comment:
Let's make it clear that you must be enrolled in automatic agent upgrades (and probably link to those docs) for this to happen.

I like to think of auth/proxy upgrades as "push" upgrades - we (Teleport) push the upgrade during the configured maintenance window. Agent upgrades are "pull" upgrades - we aren't pushing anything out, the agents are just configured to periodically check for new versions and pull them down.